tests/query/test_simple_query.py

```diff
@@ -0,0 +1,79 @@
+# --coding:utf-8--
+#
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+import sys
+from tests.common.nebula_test_suite import NebulaTestSuite
+import time
+import re
+
+
+class TestNebula(NebulaTestSuite):
+    @classmethod
+    def prepare(self):
+        resp = self.client.execute('CREATE SPACE IF NOT EXISTS space1')
+        self.check_resp_successed(resp)
+
+    def test_schema(self):
+        resp = self.client.execute('USE space1')
```

Comment:
I think we could provide a `execute` function to avoid calling `self.client` each time.